### PR TITLE
Fix failing tests

### PR DESCRIPTION
**Pull request by aaclan-ebi**
ebi-ait/dcp-ingest-central#210

### CODE REVIEW

File: tests/mocks/ingest-data/bundleManifests/example-assay-manifest.json [hunk 1]
```diff
@@ -48,14 +55,17 @@
...
     ],
     "8d6db096-f623-4e02-ab07-6aefe218c2dd": [
       "8d6db096-f623-4e02-ab07-6aefe218c2dd"
     ],
-    "6a04c2c7-aaa2-4deb-908c-15faadba8775": [
-      "6a04c2c7-aaa2-4deb-908c-15faadba8775"
+    "92300067-cf51-4ed8-95f5-cedc41ce9af1": [
+      "92300067-cf51-4ed8-95f5-cedc41ce9af1"
+    ],
+    "a26b3529-f0cf-4969-8bbc-5e6eb28ab4cc": [
```
Comment (aaclan-ebi) [inline tests/mocks/ingest-data/bundleManifests/example-assay-manifest.json:67]: analysis protocol was added

File: tests/mocks/ingest-data/bundleManifests/example-assay-manifest.json [hunk 2]
```diff
@@ -17,29 +18,35 @@
...
+    "7b90e769-c1cc-434e-a5bb-c166b4fa89df": [
+      "7b90e769-c1cc-434e-a5bb-c166b4fa89df"
     ],
     "947c0769-c1cc-434e-a5bb-c166b4fa89df": [
       "947c0769-c1cc-434e-a5bb-c166b4fa89df"
     ],
-    "7b90e769-c1cc-434e-a5bb-c166b4fa89df": [
-      "7b90e769-c1cc-434e-a5bb-c166b4fa89df"
+    "c0523a60-cb45-43fd-8718-2eaff1dcdf30": [
+      "c0523a60-cb45-43fd-8718-2eaff1dcdf30"
```
Comment (aaclan-ebi) [inline tests/mocks/ingest-data/bundleManifests/example-assay-manifest.json:35]: analysis file was added

File: tests/mocks/ingest-data/bundleManifests/example-assay-manifest.json [hunk 3]
```diff
@@ -1,11 +1,12 @@
 {
-  "envelopeUuid": "mock-submission",
   "dataFiles": [
     "uuid1",
     "uuid2",
+    "uuid",
```
Comment (aaclan-ebi) [inline tests/mocks/ingest-data/bundleManifests/example-assay-manifest.json:5]: analysis file uuid

File: tests/exporter/graph/test_graph_crawler.py [hunk 4]
```diff
@@ -29,8 +29,8 @@ def test_generate_experiment_process_graph(self):
         experiment_graph = crawler.generate_experiment_graph(test_assay_process)
 
         # then
-        self.assertEqual(len(experiment_graph.nodes.get_nodes()), 12)
-        self.assertEqual(len(experiment_graph.links.get_links()), 3)
+        self.assertEqual(len(experiment_graph.nodes.get_nodes()), 15)
+        self.assertEqual(len(experiment_graph.links.get_links()), 4)
```
Comment (aaclan-ebi) [inline tests/exporter/graph/test_graph_crawler.py:33]: analysis file, analysis protocol and analysis process was added in the nodes
and analysis process was added for the links

File: tests/mocks/ingest-data/files/mock-analysis-output-file.json [hunk 5]
```diff
@@ -27,7 +30,12 @@
   ],
   "fileName": "be53a909-2f0a-4b5c-ba67-c07f390f2f67_qc.bam",
   "cloudUrl": "s3://org-humancellatlas-upload-integration/a610450d-fcc7-49d3-a7f4-b6ed4022059b/be53a909-2f0a-4b5c-ba67-c07f390f2f67_qc.bam",
-  "checksums": null,
+  "checksums": {
+    "sha256": "sha256",
+    "crc32c": "crc32c",
+    "sha1": "sha1",
+    "s3_etag": "s3_etag"
+  },
```
Comment (aaclan-ebi) [inline tests/mocks/ingest-data/files/mock-analysis-output-file.json:38]: These are relatively newer fields that were added in the file metadata document. The test data was probably copied from an old analysis file metadata.

**Review by jacobwindsor — APPROVED** [review]

**Comment by jacobwindsor** [comment]
Thanks @aaclan-ebi. CI pipeline should catch these bugs now

**Review by MightyAx — APPROVED** [review]
LGTM

**Pull request merged**